docs/content/Switch.mdx

```diff
@@ -0,0 +1,225 @@
+  )
+
+  return (
+    <>
+      <Box display="flex" maxWidth="300px">
+        <Box flexGrow={1} fontSize={2} fontWeight="bold" id="switchLabel">
+          Notifications
+        </Box>
+        <Switch
+          aria-labelledby="switchLabel"
+          isLoading={isLoading}
+          on={isOn}
+          onClick={onClick}
+          onChange={handleSwitchChange}
+        />
+      </Box>
+      <p>The switch is {isOn ? 'on' : 'off'}</p>
+    </>
+  )
+}
+
+render(LoadingToggle)
+```
+
+### Disabled
+
+```jsx live
+<Box display="flex" maxWidth="300px">
+  <Box flexGrow={1} fontSize={2} fontWeight="bold" id="switchLabel">
+    Notifications
+  </Box>
+  <Switch aria-labelledby="switchLabel" disabled />
+</Box>
+```
+
+### With associated caption text
+
+```jsx live
+<Box display="flex">
+  <Box flexGrow={1}>
+    <Text fontSize={2} fontWeight="bold" id="switchLabel" display="block">
+      Notifications
+    </Text>
+    <Text color="fg.subtle" fontSize={1} id="switchCaption" display="block">
+      Notifications will be delivered via email and the GitHub notification center
+    </Text>
+  </Box>
+  <Switch aria-labelledby="switchLabel" aria-describedby="switchCaption" />
+</Box>
+```
+
+### Left-aligned with label
+
+```jsx live
+<>
+  <Text fontSize={2} fontWeight="bold" id="switchLabel" display="block" mb={1}>
+    Notifications
+  </Text>
+  <Switch statusLabelPosition="end" aria-labelledby="switchLabel" />
+</>
+```
+
+## Props
+
+/\*\* Whether the "on" and "off" labels should appear before or after the switch.
+
+- **This should only be changed when the switch's alignment needs to be adjusted.** For example: It needs to be left-aligned because the label appears above it and the caption appears below it.
+  \*/
+  statusLabelPosition?: 'start' | 'end'
+
+<PropsTable>
+  <PropsTableRow name="aria-describedby" type="string" description="The id of the DOM node that describes the switch" />
+  <PropsTableRow
+    name="aria-labelledby"
+    type="string"
+    required
+    description="The id of the DOM node that labels the switch"
+  />
+  <PropsTableRow name="defaultOn" type="boolean" description="Uncontrolled - whether the switch is turned on" />
+  <PropsTableRow name="disabled" type="boolean" description="Whether the switch is ready for user input" />
+  <PropsTableRow name="isLoading" type="boolean" description="Whether the switch is ready for user input" />
+  <PropsTableRow name="on" type="boolean" description="Whether the switch is turned on" />
+  <PropsTableRow
+    name="onChange"
+    type="(on: boolean) => void"
+    description="The callback that is called when the switch is toggled on or off"
+  />
+  <PropsTableRow
+    name="onClick"
+    type="(e: MouseEvent) => void"
+    description="The callback that is called when the switch is clicked"
+  />
+  <PropsTableRow name="size" type="'small' | 'medium'" description="Size of the switch" />
+  <PropsTableRow
+    name="statusLabelPosition"
+    type="'start' | 'end'"
```

Comment:
What's the default position?

Reply:
"medium". I'll add a `defaultValue` prop